tests/storage/moosefs/test_moosefs_sr.py

```diff
@@ -31,6 +31,7 @@ def test_create_moosefs_sr_without_mfsmount(self, host, moosefs_device_config):
             sr.destroy()
             assert False, "MooseFS SR creation should failed!"
 
+    @pytest.mark.usefixtures("host_no_ipv6")
```

Comment:
Everytime we use this fixture, I think there should be a comment explaining why the test has to be skipped in IPv6 mode.